Status: committed upstream after 6.3.  Fixes some terminal mangling in
gdbtui.

2004-12-04  Daniel Jacobowitz  <dan@debian.org>

	PR tui/1703
	* linux-nat.c (linux_tracefork_child): Use _exit instead of exit.
	Suggested by Joshua Neuheisel.

Index: gdb-6.3/gdb/linux-nat.c
===================================================================
--- gdb-6.3.orig/gdb/linux-nat.c	2004-12-08 18:22:04.996973094 -0500
+++ gdb-6.3/gdb/linux-nat.c	2004-12-08 18:22:20.386956067 -0500
@@ -147,7 +147,7 @@ linux_tracefork_child (void)
   ptrace (PTRACE_TRACEME, 0, 0, 0);
   kill (getpid (), SIGSTOP);
   fork ();
-  exit (0);
+  _exit (0);
 }
 
 /* Wrapper function for waitpid which handles EINTR.  */
